pkg/target/variableref_test.go

```diff
@@ -723,3 +723,81 @@ spec:
     - kustomized-nginx.example.com
 `)
 }
+
+func TestVariableRefMaps(t *testing.T) {
+	th := NewKustTestHarness(t, "/app/base")
+	th.writeK("/app/base", `
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
```

Comment:
apiVersion and kind aren't needed here for two reasons
 - the fields are optional,
 - writeK adds them anyway.

the other tests that do this are "wrong" too, but it's not an error to repeat fields :(

Reply:
right, I remove the redundancy and tests still pass 👍